Gemfile

```
@@ -1,6 +1,6 @@
 source 'https://rubygems.org/'
 
 gem 'rails-controller-testing'
-gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'spree', '~> 3.1.0'
```

Think we can revert this change since the Gemfile is only used for trying to `bundle install` inside this project.

```suggestion
gem 'spree', github: 'spree/spree', branch: 'master'
```